ckanext/switzerland/helpers/dataset_form_helpers.py

```diff
@@ -177,46 +178,64 @@ def get_contact_points_from_form(data):
 
 
 def ogdch_relations_form_helper(data):
-    """
-    sets the form field for relations
+    """Sets up the form field for relations.
     "relations": [
-    {"label": "legal_basis", "url": "https://www.admin.ch/#a20"},
-    {"label": "legal_basis", "url": "https://www.admin.ch/#a21"}]
+        {"label": {"de: "text", "fr": "text", "it": "text", "en": "text"},
+         "url": "https://www.admin.ch/#a20"},
+        ...
+    ]
     """
     relations = _get_relations_from_storage(data)
     if not relations:
         relations = get_relations_from_form(data)
 
-    data_empty = {'title': '', 'url': ''}
```

Comment:
We were using three different terms for the text of a relation in various places in the code (`label`, `title`, `text`) and it was a mess. I'm eliminating `title` to simplify things a bit.